src/services/product_services.js

```diff
@@ -1,10 +1,13 @@
 // fetching products
+import { brands } from "../temp/brands.json";
+
 export const fetchProducts = async () => {
   try {
-    const response = await (await fetch("/api/products")).json()
+    const { products } = await (await fetch("/api/products")).json();
+    const { categories } = await (await fetch("/api/categories")).json();
 
```

Comment:
Suggestion: Why have you use await twice?

Reply:
Because we have to wait for response to come from the fetch api and we have wait again to convert the response into JSON.